bin/tachyon

```diff
@@ -116,6 +117,59 @@ function runTest {
   return $RET
 }
 
+function journalCrashTest {
+  if [ "$#" -lt 1 ]; then
+    echo "Usage: tachyon journalCrashTest <test-duration-in-seconds> [-options]"
```

Comment:
Is there a way to leverage the commons-cli library to print out the usage automatically? Having it print out in both this file and the .java file seems error prone to keep consistent.

Also, why is the test duration not in the cli options?